_posts/2017-10-20-the-data-and-science-behind-grabshare-part-i.md

```diff
@@ -0,0 +1,117 @@
+
+<table class="center">
+  <tr>
+    <td>Hours </td>
+    <td>8-10</td>
+    <td>10-13</td>
+    <td>14-16</td>
+    <td>16-18</td>
+    <td>18-22</td>
+    <td>Others</td>
+  </tr>
+  <tr>
+    <td>Coverage</td>
+    <td>46%</td>
+    <td>39%</td>
+    <td>35%</td>
+    <td>38%</td>
+    <td>43%</td>
+    <td>22%</td>
+  </tr>
+</table>
+
+<div class="table-caption">Table 1. Cluster coverage of different time periods</div>
+
+The assignment flow of typical GC bookings is stated in Algorithm I. For every newly arrived booking, we search for nearby drivers and check for their availability condition. If no driver is available, we recycle it to the next round of assignment. Otherwise we select the most suitable driver for them. Leveraging the current system structure, we planned to extend the GrabCar service to GrabShare by maintaining more detailed bookings and driver state information along with an additional check on seat reservation. 
+
+![GrabCar booking assignment flow](/img/the-data-and-science-behind-grabshare-part-i/grabcar-booking-assignment-flow.png)
+*Algorithm I. GrabCar booking assignment flow.*
+
+Specifically, Algorithm II gives the assignment flow of Grab- Share bookings. We can see that its overall structure is the same with GrabCar except for two differences. Firstly, the candidate driver set is different. For every new GrabShare booking, we search for in-transit GrabShare drivers who are currently serving at least one GrabShare booking. Therefore, we need to check seat availability condition to ensure that the vehicle has enough remaining seats to serve the new GrabShare booking. Mathematically, the following seat reservation constraint needs to be satisfied for a successful assignment between booking bki and driver drj: 
+
+![constraint](/img/the-data-and-science-behind-grabshare-part-i/constraint.png)
+
+where s(drj) denotes the total capacity of the vehicle drj, op(drj) is one of the maintained variable that denotes the current occupied capacity of the vehicle drj and rp (bki) is the required capacity for booking bki. To make it consistent, we also need to update the vehicle occupied capacity variable op (drj) by adding the booking required capacity rp (bki) after every successful assignment or removing it if cancellation occurs.
+
+![GrabShare booking assignment flow](/img/the-data-and-science-behind-grabshare-part-i/grabshare-booking-assignment-flow.png)
+*Algorithm II. GrabShare booking assignment flow*
```

Comment:
For image captions, they have been traditionally done like this:

```
<div class="post-image-section">
  <img alt="Corgi Macbook" src="/img/ios-automation/corgi-macbook-meme.jpg" width="80%">
  <small class="post-image-caption">Literally anyone</small>
</div>
```

Reply:
I think you can do the same for table captions. Don't have to add a new class.

Reply:
okay will use that format then.